bundles/org.openhab.binding.automower/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -164,4 +179,64 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="plannerNextStartTimestampType">
+		<item-type>DateTime</item-type>
+		<label>Next Auto Start</label>
+		<description>The channel providing the time for the next auto start. If the mower is charging then the value is the
+			estimated time when it will be leaving the charging station. If the mower is about to start now, the value is NULL.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="plannerOverrideActionType">
+		<item-type>String</item-type>
+		<label>The override action</label>
+		<description>The channel providing an action that overrides current planner operation.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="calendarTasksType">
+		<item-type>String</item-type>
+		<label>The information about the planner as JSON</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
done